spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java

```diff
@@ -137,533 +80,107 @@ public Streams getStreams() {
 		return this.streams;
 	}
 
-	public Ssl getSsl() {
-		return this.ssl;
-	}
-
 	public Jaas getJaas() {
 		return this.jaas;
 	}
 
-	public Template getTemplate() {
-		return this.template;
+	public Reactor getReactor() {
+		return this.reactor;
 	}
 
-	public Security getSecurity() {
-		return this.security;
+	public Template getTemplate() {
+		return this.template;
 	}
 
 	public Retry getRetry() {
 		return this.retry;
 	}
 
-	private Map<String, Object> buildCommonProperties() {
-		Map<String, Object> properties = new HashMap<>();
-		if (this.bootstrapServers != null) {
-			properties.put(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG, this.bootstrapServers);
-		}
-		if (this.clientId != null) {
-			properties.put(CommonClientConfigs.CLIENT_ID_CONFIG, this.clientId);
-		}
-		properties.putAll(this.ssl.buildProperties());
-		properties.putAll(this.security.buildProperties());
-		if (!CollectionUtils.isEmpty(this.properties)) {
-			properties.putAll(this.properties);
-		}
-		return properties;
-	}
-
 	/**
-	 * Create an initial map of consumer properties from the state of this instance.
+	 * Create an initial map of admin properties from the state of this instance.
 	 * <p>
 	 * This allows you to add additional properties, if necessary, and override the
-	 * default kafkaConsumerFactory bean.
-	 * @return the consumer properties initialized with the customizations defined on this
+	 * default kafkaAdmin bean.
+	 * @return the admin properties initialized with the customizations defined on this
 	 * instance
 	 */
-	public Map<String, Object> buildConsumerProperties() {
-		Map<String, Object> properties = buildCommonProperties();
-		properties.putAll(this.consumer.buildProperties());
+	public Map<String, Object> buildAdminProperties() {
+		// spring.kafka.<common-props>
```

Comment:
ℹ️ will probably yank these prefix/path comments out later - just easily shows what the mapping is for this proposal review only. It may be helpful to move them up to the method javadoc though in some capacity.